### PR TITLE
SCMOD-5524: Using com.github.workerframework.testing

### PR DESCRIPTION
**Pull request by xbreizh**
https://portal.digitalsafe.net/browse/SCMOD-5524

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -116,6 +116,16 @@
...
                 <version>3.3.0-SNAPSHOT</version>
             </dependency>
+            <dependency>
+                <groupId>com.github.workerframework.testing</groupId>
+                <artifactId>workerframework-testing-util</artifactId>
+                <version>1.0.0-SNAPSHOT</version>
+            </dependency>
+            <dependency>
+                <groupId>com.github.workerframework.testing</groupId>
+                <artifactId>workerframework-testing-integration</artifactId>
```
Comment (dermot-hardy) [inline pom.xml]: Dependency management should be in alphabetical order.

File: worker-markup-container-fs/pom.xml [hunk 2]
```diff
@@ -58,9 +58,8 @@
             <scope>runtime</scope>
         </dependency>
         <dependency>
-            <groupId>com.github.workerframework</groupId>
-            <artifactId>worker-testing-integration</artifactId>
-            <scope>test</scope>
```
Comment (dermot-hardy) [inline worker-markup-container-fs/pom.xml:63]: Why are we dropping test scope?

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (xbreizh) [reply pom.xml]: re-ordered

File: worker-markup-container-fs/pom.xml (hunk 2 above)
Comment (xbreizh) [reply worker-markup-container-fs/pom.xml:63]: restored

**Pull request merged**